### DISCUSSION TITLE
Different suicide commands

### DISCUSSION THREAD

**Discussion by Fogapod** [unitystation/unitystation#7518] [Ideas]
### Description
I think having multiple ways to die could be good for roleplay. Also I don't like word `suicide`.

These are commands existing in Team Fortress 2:
- `kill`: your soul departures body and body drops dead.
- `explode`: your body explodes into pieces, similar to blob, gibs have high velocity and do a bit of harm on impact, leaving blood on the floor. clowns could drop a bunch of funny stuff instead of their organs (TF2 pyrovision).

Not sure what other commands could be added yet.

Maybe in future these commands could accept arguments to let admins target users.

**Comment by Pockets-byte** [comment]
in ss13 your suicide depends on the item you're holding

**Comment by PerfectTangent** [comment]
converting to discussion
